CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ so fresh eyes are always welcome.
 ## Using GitHub
 
 If you choose to contribute via GitHub, you may want to look at
-[How to Contribute to an Open Source Project on GitHub][how-contribute].
```

Comment:
`][how-contribute].`... this is an alias for a link. The actual link is at the end of this document, along with several other links. That's useful in case we need to use the link in multiple places (but update a single place) and also to reduce cluttering in the source.